### DISCUSSION TITLE
Bug in the wallpaper "The hill"

### DISCUSSION THREAD

**Discussion by DodgyX** [rocksdanister/lively#1401] [General]
I just discovered a bug in the wallpaper "The hill" (just look at the time, its not in the center...)
<img width="960" alt="Capture" src="https://user-images.githubusercontent.com/78494293/191812416-81c80d8a-1023-49d7-946a-877f729ffc4a.PNG">

**Comment by rocksdanister** [comment]
@dabjulmaros FYI 😀
